samples/azure-quantum/variational-quantum-eigensolver/VQE-qiskit-hydrogen-session.ipynb

```diff
@@ -0,0 +1,416 @@
+        "from qiskit_nature.second_q.circuit.library import UCCSD\n",
+        "\n",
+        "problem = get_problem()\n",
+        "qubit_converter = get_qubit_converter()\n",
+        "\n",
+        "spsa_optimizer=SPSA(maxiter=100)\n",
+        "\n",
+        "noiseless_estimator = AerEstimator(\n",
+        "    run_options={ \"shots\": 1000 }\n",
+        ")\n",
+        "\n",
+        "vqe_solver = VQEUCCFactory(estimator=noiseless_estimator, ansatz=UCCSD(), optimizer=spsa_optimizer)\n",
+        "calc = GroundStateEigensolver(qubit_converter, vqe_solver)\n",
+        "result = calc.solve(problem)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "print(\"Local AER simulator result:\\n\")\n",
+        "print(result.groundenergy)"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "editable": true,
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "run_control": {
+          "frozen": false
+        }
+      },
+      "source": [
+        "## Run on an Azure Quantum backend\n",
+        "Now, you will run the same problem on a backend of your choice through Azure Quantum. Here the maximum number of iterations of the optimizer is set to 50.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "from azure.quantum.qiskit import AzureQuantumProvider\n",
+        "\n",
+        "# Connect to the Azure Quantum workspace via a Qiskit provider\n",
+        "provider = AzureQuantumProvider(\n",
+        "            resource_id = \"\",\n",
+        "            location = \"\")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "ionq_sim = provider.get_backend('ionq.simulator')\n",
+        "quantinuum_sim = provider.get_backend('quantinuum.sim.h1-1e')\n",
+        "rigetti_sim = provider.get_backend('rigetti.sim.qvm')\n",
+        "\n",
+        "# Set the backend you want to use here.\n",
```

Comment:
```suggestion
        "# Set the backend you want to use here.\n",
        "# WARNING: Quantinuum emulator usage is not unlimited. Running this sample against it could consume a significant amount of your eHQC quota.\n"
```